gcloud/bigquery/table.py

```diff
@@ -790,3 +778,18 @@ def _string_from_json(value, _):
     'RECORD': _record_from_json,
     'STRING': _string_from_json,
 }
+
+
+def _rows_from_json(rows, schema):
```

Comment:
Maybe.  I moved it out to a separate function to allow calling from `RunSynchQueryJob.fetch_data`.